docs/core/v3.0.rst

```diff
@@ -1160,8 +1158,8 @@ The following members are optional:
     transformer is used, same for an empty list.
 
 
-All other names within the array metadata object are reserved for
-future versions of this specification.
+The array metadata object must not contain any other names.
```

Comment:
This could be clarified to state that that an implementation MUST fail to open a zarr repo/array with unknown metadata fields; that would allow required extensions to be defined in a more concise way.

Reply:
> that would allow required extensions to be defined in a more concise way.

Not sure what you mean here, what would change regarding required extensions?

Anyways, I'll add this.